_posts/2024-03-15-customer-experience-outcomes.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: post
+title:  "Customer experience outcomes of focused, user-centered product design improvements"
+authors:
+  - apasun
+date:   2024-03-15 0:00:10 -0700
+categories:
+  - community
+excerpt: The OpenSearch Project invites the OpenSearch community to explore the future of search, analytics, and generative AI at the first OpenSearch user conference in Europe. Join us in Berlin on May 6 & 7 and learn how to build powerful applications and get the most out of your OpenSearch deployments.
+meta_keywords: 
+meta_description: 
+---
+
+## User experience challenges
+
+A large proportion of software improvements focus on features and functionality. It is common practice for product managers and developers to incrementally add engineering capabilities. When the implementation of a seamless experience is not taken into account, the result can be a fragmented user experience. While these changes may not affect the perceived experience of a high-code user, low-code users report excess frustration and a lack of satisfaction with an experience when changes to software do not aid in accomplishing set outcomes that a software experience intends to deliver.
```

Comment:
```suggestion
Software improvements often prioritize adding new features and functionality. Product managers and developers often adopt an agile approach, adding engineering capabilities in iterative steps. Neglecting seamless user experience can lead to a fragmented journey, especially for low-code users. While high-code users might adapt, low-code users often report frustration and dissatisfaction when software updates don't actually help them achieve their goals.
```

Reply:
Global comments: Sentences should be succinct, typically no more than 25 words in a sentence. Active voice is preferred, especially in blogs. Revisions made to emphasize the message I think you're trying to send and to grab the reader's attention.

Reply:
I'm not sure how the thought in the first two sentences ties into the user experience. I suggest reworking to tie those concepts together.